src/bloqade/submission/quera.py

```diff
@@ -42,27 +73,80 @@ def queue_api(self):
         return self._queue_api
 
     def get_capabilities(self, use_experimental: bool = False) -> QuEraCapabilities:
+        """Get the capabilities of the QuEra backend.
+
+        Args:
+            use_experimental (bool): Whether to use experimental capabilities of
+                the QuEra system. Defaults to `False`.
+
+        Returns:
+            capabilities (QuEraCapabilities): capabilities of the selected
+                QuEra backend.
+        """
         try:
             return QuEraCapabilities(**self.queue_api.get_capabilities())
         except BaseException:
             return super().get_capabilities(use_experimental)
 
     def submit_task(self, task_ir: QuEraTaskSpecification) -> str:
+        """Submit the task to the QuEra backend.
+
+        Args:
+            task_ir (QuEraTaskSpecification): task IR to be
+                executed on the QuEra backend.
+
+        Returns:
+            task_id (str): Task id as a result of executing
+                IR on the QuEra backend.
+        """
         return self.queue_api.submit_task(
             task_ir.json(by_alias=True, exclude_none=True, exclude_unset=True)
         )
 
     def task_results(self, task_id: str) -> QuEraTaskResults:
+        """Get the status of the task submitted to the QuEra backend
```

Comment:
```suggestion
        """Get the status of a task previously submitted to the QuEra backend
```

Just add more context about when this will be executed. For example here you can call this method on any previously submitted task so this wording is more appropriate. 

I think you use this language throughout the documentation so please update the language on those functions as well.